### ISSUE TITLE
Building curl for arm64-v8a with openssl causes error:

### ISSUE THREAD

**Issue by rhardih** [rhardih/bad#14]
```bash
root@e386d3d6b859:/curl-7.64.0# make
Making all in lib
make[1]: Entering directory '/curl-7.64.0/lib'
make  all-am
make[2]: Entering directory '/curl-7.64.0/lib'
make[2]: Nothing to be done for 'all-am'.
make[2]: Leaving directory '/curl-7.64.0/lib'
make[1]: Leaving directory '/curl-7.64.0/lib'
Making all in src
make[1]: Entering directory '/curl-7.64.0/src'
Making all in ../docs
make[2]: Entering directory '/curl-7.64.0/docs'
Making all in .
make[3]: Entering directory '/curl-7.64.0/docs'
make[3]: Nothing to be done for 'all-am'.
make[3]: Leaving directory '/curl-7.64.0/docs'
Making all in cmdline-opts
make[3]: Entering directory '/curl-7.64.0/docs/cmdline-opts'
make[3]: Nothing to be done for 'all'.
make[3]: Leaving directory '/curl-7.64.0/docs/cmdline-opts'
make[2]: Leaving directory '/curl-7.64.0/docs'
make[2]: Entering directory '/curl-7.64.0/src'
  CCLD     curl
/android-23-toolchain/bin/../lib/gcc/aarch64-linux-android/4.9.x/../../../../aarch64-linux-android/bin/ld: warning: libssl.so.1.1, needed by ../lib/.libs/libcurl.so, not found (try using -rpath or -rpath-link)
/android-23-toolchain/bin/../lib/gcc/aarch64-linux-android/4.9.x/../../../../aarch64-linux-android/bin/ld: warning: libcrypto.so.1.1, needed by ../lib/.libs/libcurl.so, not found (try using -rpath or -rpath-link)
../lib/.libs/libcurl.so: undefined reference to `DES_ecb_encrypt@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `ERR_clear_error@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `EVP_md5@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `UI_method_set_closer@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `RAND_load_file@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `DES_set_key@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_version@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `OpenSSL_version_num@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `EVP_sha256@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `RSA_get0_key@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `X509_get_issuer_name@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `X509_free@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `ERR_error_string_n@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_CTX_use_certificate_file@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `d2i_OCSP_RESPONSE@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `OCSP_RESPONSE_free@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `EVP_DigestInit_ex@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `d2i_PKCS12_bio@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `UI_method_set_writer@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `OPENSSL_sk_value@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `X509_EXTENSION_get_object@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_CTX_get_cert_store@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `CONF_modules_load_file@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_CTX_use_certificate_chain_file@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `ENGINE_get_id@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `EVP_PKEY_free@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `UI_method_set_reader@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `X509V3_EXT_print@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_get0_alpn_selected@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `X509_get0_signature@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `X509_check_issued@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `RSA_free@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_CTX_set_default_passwd_cb_userdata@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `BIO_printf@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_set_ex_data@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `RAND_bytes@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `EVP_MD_CTX_new@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `i2t_ASN1_OBJECT@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_CTX_sess_set_new_cb@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_CTX_set_cipher_list@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `ASN1_STRING_to_UTF8@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `X509_LOOKUP_file@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `X509_get0_notBefore@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `DH_get0_key@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `ASN1_STRING_new@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `MD5_Init@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `OPENSSL_sk_pop_free@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `ENGINE_load_private_key@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_write@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `X509_NAME_get_entry@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_SESSION_free@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `ASN1_STRING_get0_data@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_CTX_set_post_handshake_auth@OPENSSL_1_1_1'
../lib/.libs/libcurl.so: undefined reference to `SSL_get_certificate@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `PEM_write_bio_X509@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `X509_load_crl_file@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `X509_signature_print@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_CTX_ctrl@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `OCSP_basic_verify@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `ENGINE_init@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `OCSP_response_get1_basic@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `X509_get_serialNumber@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `ASN1_STRING_free@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `BIO_free@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `EVP_DigestFinal_ex@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_CTX_use_PrivateKey_file@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_pending@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `X509_get_subject_name@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `i2a_ASN1_OBJECT@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_CTX_set_default_passwd_cb@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_read@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `UI_get_input_flags@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `MD5_Final@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `RSA_flags@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `ENGINE_get_first@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `X509_get_pubkey@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `BIO_ctrl@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `UI_get_string_type@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_get_peer_cert_chain@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `OPENSSL_sk_num@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `RAND_file_name@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `UI_get0_user_data@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_free@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `GENERAL_NAMES_free@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `EVP_PKEY_get0_RSA@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `PKCS12_PBE_add@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `i2d_X509_PUBKEY@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SHA256_Update@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `X509_verify_cert_error_string@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `OCSP_single_get0_status@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `ENGINE_get_next@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `UI_set_result@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_get_error@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_get_current_cipher@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `BIO_puts@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `EVP_PKEY_get0_DSA@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `PKCS12_parse@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `CRYPTO_get_ex_new_index@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `UI_destroy_method@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `X509_get0_notAfter@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `UI_method_get_opener@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `OCSP_check_validity@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_set_fd@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `X509_STORE_add_lookup@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_get_ex_data@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `MD4_Final@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_CTX_use_PrivateKey@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `BIO_new@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_get_peer_certificate@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_CTX_set_srp_password@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `MD4_Update@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SHA256_Final@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_CTX_set_msg_callback@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `OCSP_cert_status_str@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_CTX_set_next_proto_select_cb@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_alert_desc_string_long@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `EVP_PKEY_get1_RSA@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `DSA_get0_key@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `EVP_MD_CTX_free@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `EVP_PKEY_copy_parameters@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_set_session@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_ctrl@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `DSA_get0_pqg@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_CTX_check_private_key@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `X509_NAME_print_ex@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `UI_method_get_reader@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `ASN1_STRING_length@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `OPENSSL_load_builtin_modules@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `UI_method_set_opener@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `ENGINE_set_default@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `EVP_PKEY_get0_DH@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `PKCS12_free@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `ENGINE_load_builtin_engines@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_shutdown@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `DH_get0_pqg@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `CRYPTO_malloc@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `ENGINE_by_id@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_CTX_add_client_CA@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `PEM_read_bio_X509@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `CRYPTO_free@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `OCSP_resp_get0@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_get_verify_result@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_CTX_use_certificate@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `MD5_Update@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_get_shutdown@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `BN_print@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `ENGINE_free@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `UI_create_method@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `ERR_peek_error@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `OCSP_response_status@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `DES_set_odd_parity@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_connect@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `UI_OpenSSL@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_CTX_set_alpn_protos@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_get_privatekey@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `OCSP_crl_reason_str@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `X509_NAME_ENTRY_get_data@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `EVP_PKEY_id@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_CTX_set_srp_username@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `X509_NAME_get_index_by_NID@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `BIO_s_mem@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `OCSP_resp_count@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_set_connect_state@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `X509_get_version@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_CTX_set_options@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_CTX_set_verify@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `OCSP_response_status_str@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `UI_method_get_closer@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `OPENSSL_sk_pop@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `MD4_Init@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_CTX_free@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `ASN1_STRING_type@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `BIO_f_ssl@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_new@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `X509_EXTENSION_get_data@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `X509_get_X509_PUBKEY@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `RAND_status@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `TLS_client_method@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `BIO_s_file@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_CTX_set_ciphersuites@OPENSSL_1_1_1'
../lib/.libs/libcurl.so: undefined reference to `X509_STORE_set_flags@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `RAND_add@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `EVP_DigestUpdate@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SHA256_Init@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `ENGINE_ctrl_cmd@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `X509_get_ext_d2i@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `X509_get0_extensions@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_CIPHER_get_name@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `ENGINE_finish@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `ERR_get_error@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `OCSP_BASICRESP_free@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `ASN1_TIME_print@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `ASN1_STRING_print@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_set_bio@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_CTX_set_keylog_callback@OPENSSL_1_1_1'
../lib/.libs/libcurl.so: undefined reference to `SSL_CTX_new@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `ENGINE_ctrl@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `SSL_CTX_load_verify_locations@OPENSSL_1_1_0'
../lib/.libs/libcurl.so: undefined reference to `UI_method_get_writer@OPENSSL_1_1_0'
clang70: error: linker command failed with exit code 1 (use -v to see invocation)
Makefile:913: recipe for target 'curl' failed
make[2]: *** [curl] Error 1
make[2]: Leaving directory '/curl-7.64.0/src'
Makefile:1958: recipe for target 'all-recursive' failed
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory '/curl-7.64.0/src'
Makefile:920: recipe for target 'all-recursive' failed
make: *** [all-recursive] Error 1
```

Branch with modified files here:

https://github.com/rhardih/bad/compare/curl-arm64-with-openssl

**Comment by ArtificialRaccoon** [comment]
It's been a while since we talked about this, but I have returned to this work and may have solved it.
In the curl dockerfile, we attempt to set the LDFLAGS environment variable, however it appears to have been ignored.

I was reading the cURL build docs (https://curl.se/docs/install.html) and noticed that they appear to set the LDFLAGS on the same line as the call to configure.  I did exactly that, and the build completed successfully.  My dockerfile thus has a command like this:
```
RUN LDFLAGS=-Wl,-R/openssl-build/lib ./configure \
    --with-ssl=/openssl-build \
    --host=$HOST\
    --prefix=/curl-build/
```

I still need to test on hardware, but it does appear to have completed successfully.

**Comment by ArtificialRaccoon** [comment]
Tested on Android 9.0, ARM64-v8a device (Sony G8441).  Device was able to connect to WMS and WMTS Endpoints over HTTPS, and were able to successfully pull data.  Looks like the solution was just that simple.

**Comment by rhardih** [comment]
Great! Would you mind creating a pull request with your change?

**Comment by ArtificialRaccoon** [comment]
@rhardih Done.  Sorry I took so long; wasn't working with Android for the last month.

Cheers

**Comment by rhardih** [comment]
Fixed by #16.
